conn.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	ggio "github.com/gogo/protobuf/io"
 	inet "github.com/libp2p/go-libp2p-net"
-	peer "github.com/libp2p/go-libp2p-peer"
+	"github.com/libp2p/go-libp2p-peer"
```

Comment:
don't remove this.

Reply:
I can replace all of the explicit import names, but they are redundant to the exported package names, which is why my IDE deleted them. Is there a description or convention on which imports get names and which do not?

Reply:
convention in the comment below.

Reply:
Thank you